test/util.js

```diff
@@ -16,7 +15,7 @@ function gen()
 
 module.exports = {
   init: function(options) {
-    var store = racer.createStore({db: new Memory()});
+    var store = racer.createBackend();
```

Comment:
For consistency, might it be better to rename this variable to `backend` rather than `store`?